src/pivotal-ui/components/typography/typography.scss

```diff
@@ -113,6 +113,12 @@ If it's not a heading but you need similar visual treatment you can add just the
 // Override the bootstrap margins to use our margin classes so we can align
 // things. This is necessary because we have to keep the whitespace margins in sync with this
 
+h1, h2, h3, h4, h5, h6,
+.h1, .h2, .h3, .h4, .h5, .h6 {
+  padding-top: 0.107em;
```

Comment:
Let's make this into a variable in pui-variables.scss, ideally something that describes the math you did to get to this number. e.g. calculated in terms of the new line-height value?